packages/agreement-process/test/activateAgreement.test.ts

```diff
@@ -475,7 +475,7 @@ describe("activate agreement", () => {
         ).toContain(expectedContract.path);
 
         await testRelatedAgreementsArchiviation(relatedAgreements);
-        expect(acrivateAgreementReturnValue).toMatchObject(
+        expect(activateAgreementReturnValue).toMatchObject(
           expectedActivatedAgreement
         );
```

Comment:
Here and in all other tests you modified we could instead match the entire returned payload including metadata - so we follow the "standard" of matching the entire object returned by service functions. What do you think?
I propose to do the same for all other service methods we will modify in API v2 developments

```suggestion
        expect(activateAgreementReturnValue).toMatchObject({
          data: expectedActivatedAgreement,
          metadata: {
             version: 1
          },
        );
```

Reply:
I think it would make sense, that way we'll also test the behavior that we're expecting in the new m2m-gateway.
I'll add that to the tests I've changed here